src/itest/java/uk/gov/companieshouse/charges/data/controller/HealthCheckControllerITest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.charges.data.controller;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
```

Comment:
We can probably delete this iTest and other sample iTests for now, and Cucumber story should add a sample iTest in BDD style and all iTests from then should follow same style

Reply:
ok, will delete it